.github/workflows/update-progress.yml

```
@@ -0,0 +1,47 @@
+          if git diff --cached --quiet; then
+            echo "changed=false" >> $GITHUB_OUTPUT
+          else
+            git commit -m "ci: update progress bars and KLoC [skip ci]"
+            echo "changed=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Push commit
+        if: steps.commit.outputs.changed == 'true'
+        run: |
+          git push
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)